### DISCUSSION TITLE
[Questions]  Any way to export dashboard settings ?  i.e. visible column config?

### DISCUSSION THREAD

**Discussion by tracure1337** [rabbitmq/rabbitmq-server#12967] [Questions] [answered]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)
- [X] I run RabbitMQ 4.x, the only series currently covered by [community support](https://www.rabbitmq.com/release-information)
- [X] I promise to provide all relevant information (versions, logs from all nodes, rabbitmq-diagnostics output, detailed reproduction steps)

### RabbitMQ version used

4.0.4

### Erlang version used

26.2.x

### Operating system (distribution) used

Rocky

### How is RabbitMQ deployed?

Community Docker image

### What problem are you trying to solve?

I would like tto export the column specific settings to a file or have it in the definition so it's automatically configured the same on every deplyment. 

![image](https://github.com/user-attachments/assets/cd7c3aa6-c759-4a2e-92ee-058b91285e4f)
For example client, connected_at and SSL/TLS as in the example.

**Comment by lukebakken** [comment, marked as answer]
Hello, thanks for using RabbitMQ. The management interface does not support this feature, but it's open-source, so someone could contribute it if they'd like.

**Reply by tracure1337** [reply]
Thank you!

**Comment by michaelklishin** [comment]
@cocoonkid those settings are stored in the client-local (that is, browser/site-local) cache. Developer Tools might have a way to export and import those.

As already explained by Luke, the management plugin does not have this feature and this must be the 2nd time I see it mentioned in the ≈ 15 year history of that plugin.

**Reply by tracure1337** [reply]
Thank you for taking the time :-)
